content/docs/get-started-with-neon/get-started-branching.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Get started with Branching
+enableTableOfContents: true
+isDraft: false
+---
+
+This topic describes how to get started with Neon's branching feature, which allows you to branch your data in the same way that you branch your code.
+
+A branch is a clone of your data that you are free to play around with and modify without affecting the originating data.
+For more information about what branches are and how to use them in your development workflows, see [Branching](../../conceptual-guides/branching).
+
+_Neon Branching capabilities are not yet publicly available. If you would like to try this feature, reach out to [iwantbranching@neon.tech](mailto:iwantbranching@neon.tech) describing your use case and requesting that Neon enable branching for your account._
+
+You can create and manage branches using the Neon Console or Neon API. In this topic, we cover branching using the Neon Console.
+
+Before you can create a branch, you must have a Neon project to branch from. If you do not have a Neon project, see [Setting up a project](../setting-up-a-project).
+
+## Create a branch
+
+To create a branch:
+
+1. On the Neon Console, select a Neon project.
+2. Select the **Branches** tab.
+3. Click **New Branch** to open the branch creation dialog.
+![Create branch dialog](./images/create_branch.png)
+4. Enter a name for the branch.
+5. Select a parent branch. You can branch from your Neon project's `main` branch or a previously created branch. Every Neon project is created with a default branch called `main`.  
+6. Select one of the following branching options:
+    - **Head**: Creates a branch with data up to the current point in time (the default).
+    - **Time**: Creates a branch with data up to the specified date and time.
+    - **LSN**: Creates a branch with data up to the specified [Log Sequence Number (LSN)](../../reference/glossary/#lsn).
+7. Click **Create Branch** to create your branch.
+
+You are directed to the **Branches** tab where you are shown the details for your new branch.
+
+## View branches
+
+To view the branches in a Neon project:
+
+1. In the Neon Console, select a project from the project drop-down list.
```

Comment:
```suggestion
1. In the Neon Console, select a project from the project list.
```
we're considering getting rid of that dropdown, so it's best to refer to the list of the projects that's not going anywhere